src/Service/Modules/SaHoldStoriesModule.cs

```diff
@@ -72,14 +72,14 @@ private object GetHoldStoriesForRootProduct(string rootProduct)
         private object UpdateSaHoldStory(int holdStoryId)
         {
             this.RequiresAuthentication();
-            var privileges = this.Context.CurrentUser.GetPrivileges().ToList();
+            var privileges = this.Context?.CurrentUser?.GetPrivileges().ToList();
 
             if (!this.authorisationService.HasPermissionFor(AuthorisedAction.ProductHold, privileges))
             {
                 return this.Negotiate.WithModel(new UnauthorisedResult<SaHoldStory>("You are not authorised to update hold stories."));
             }
 
-            var employeeUri = this.Context.CurrentUser.GetEmployeeUri();
+            var employeeUri = this.Context?.CurrentUser?.GetEmployeeUri();
```

Comment:
I think in the update and add methods it probably should fail if the employee uri isn't available?